web/third/bootstrap-typeahead/typeahead.js

```diff
@@ -477,7 +477,10 @@ import {get_string_diff} from "../../src/util";
         default:
           var hideOnEmpty = false
           if (e.keyCode === 8 && this.options.helpOnEmptyStrings) { // backspace
-            hideOnEmpty = true
+            // If hideOnEmpty is explicitly set to false, we don't want to hide.
+            if(typeof(this.options.hideOnEmpty) == undefined || this.options.hideOnEmpty){
```

Comment:
Always use `===` in JavaScript code.

Reply:
This has a bunch of other style issues; the linter would catch this if this file weren't part of Bootstrap.